apps/sms/index.html

```diff
@@ -84,6 +85,21 @@
     <script defer src="services/js/threads.js"></script>
     <script defer src="views/inbox/js/inbox.js"></script>
     <script defer src="views/shared/js/settings.js"></script>
+    <script defer src="lib/bridge/bridge.js"></script>
+    <script defer src="lib/bridge/plugins/stream/client.js"></script>
+    <script defer src="lib/bridge/plugins/stream/service.js"></script>
+    <script defer src="lib/bridge/plugins/finalize/client.js"></script>
+    <script defer src="services/js/bridge_service_mixin.js"></script>
+    <script defer src="services/js/moz_mobile_connections/moz_mobile_connections_shim.js"></script>
+    <script defer src="services/js/moz_mobile_message/moz_mobile_message_shim.js"></script>
+    <script defer src="services/js/moz_settings/moz_settings_shim.js"></script>
+    <script defer src="services/js/moz_mobile_message/moz_mobile_message_client.js"></script>
+    <script defer src="services/js/conversation/conversation_service.js"></script>
+    <script defer src="services/js/conversation/conversation_client.js"></script>
+    <script defer src="services/js/messaging/messaging_service.js"></script>
```

Comment:
I don't think it will bring obvious penalty since we'll uglyfy and concatenate the scripts in production build. Just wondering it's just for or we'll need the messaging service in the main startup call path in this patch?(sorry I didn't read whole patch but I can find why we must have this in window context)